src/styles/components/main.scss

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
```

Comment:
It is worth avoiding working with tags directly and setting styles more than classes.